ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -556,11 +556,11 @@ private static void RemoveDuplicateCountries(ITravelOrVisitorDetails details)
 
         private async Task<PatientDetails> ExtractPatientDetails(MigrationDbNotification notification)
         {
-            var addressRaw = string.Join(" \n",
-                notification.Line1,
-                notification.Line2,
-                notification.City,
-                notification.County);
+            var addressLines = new List<string>
+            {
+                notification.Line1, notification.Line2, notification.City, notification.County
+            };
+            var addressRaw = string.Join(" \n", addressLines.Where(line => !string.IsNullOrEmpty(line)));
```

Comment:
Do you know why we're using `" \n"` rather than just `"\n"` here? It seems weird that we'd want to add a space to the end of each line.

Reply:
Not sure, I have removed it and also trimmed the inputs